CDCgov/prime-reportstream
# experience/7308

Error messages were not showing up properly in the Message Tracker Message details page. This fixes the problem


**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. run the backend
2. make sure the test message data is in the database (or at least messages with warnings and errors)
3. log in to ReportStream's front-end as a PrimeAdmin roled user
4. click on the "Message Id Search" link in the Admin dropdown
5. (if test data) search for "Hawaii"
6. click on one of the results that probably maybe has an error or warning for the message ("Hawaii6" is a good one)
7. Scroll allll the way down to the bottom of the page to the "Warning" and "Error" section
8. Result: the warning/error table should be populated now

## Changes
### Before:
![image](https://user-images.githubusercontent.com/19178435/200894499-b13a98b0-021e-4a91-9466-1806b4a58906.png)


### After:
![image](https://user-images.githubusercontent.com/19178435/200894937-11f580ab-5aa3-4cc6-8178-a704ef07dcca.png)


## Checklist

### Testing

### Process

## Linked Issues
- Fixes #7308 

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
